index.html

```diff
@@ -4,19 +4,33 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>301 Lab 1</title>
+
+  <link rel="stylesheet" type="text/css" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/base.css" type="text/css">
+  <link rel="stylesheet" href="./css/layout.css" type="text/css">
+  <link rel="stylesheet" href="./css/module.css" type="text/css">
+
 </head>
 
 <body>
 
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
+  <section id="mainContainer">
+    <div id="a"></div>
+    <section id="sideBar">
```

Comment:
Overall, you can consider using the more semantic tags such as <header>, <main>, <footer>, <aside>, <section>, <article>, etc. Think about how this page could be organized, and how you could target the desired elements in a way that does not depend so heavily on the <div> tags and id properties.